backend/persistent/datastores/drivers/AWSAthenaDriver.js

```diff
@@ -0,0 +1,222 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+import Logger from '../../../logger';
```

Comment:
I wouldn't use `Logger` in this file.
I'd let errors propagate up to the connector.
Otherwise an error would be logged multiple time (here in the driver and in the connector).